sdk/cloudmachine/Azure.Provisioning.CloudMachine/src/FeaturesBuiltIn/AIFoundryFeature.cs

```diff
@@ -69,8 +83,23 @@ protected internal override void EmitConnections(ConnectionCollection connection
         /// <returns>A placeholder or no-op resource, as provisioning is out-of-band for now.</returns>
         protected override ProvisionableResource EmitResources(ProjectInfrastructure cm)
         {
-            // In the future, generate real provisioning resources for Foundry
-            return new NoOpResource();
+            var cmId = cm.Id;
+            AIFoundryCognitiveServiceCdk cs = new($"{cmId}_foundry", $"{cmId}-foundry");
```

Comment:
why not consistently `{cmId}_foundry` ?

Reply:
we use underscores in other places for bicep IDs. But foundry resource name cannot have underscores.